packages/api/.env

```diff
@@ -1 +1 @@
-MONGODB_URI=mongodb://127.0.0.1:27017/gloss-translation
\ No newline at end of file
+DATABASE_URL=
```

Comment:
Am I supposed to leave `.env` with these contents? I created a `.env.local` file with this content:  
```
DATABASE_URL=localhost:5432
```
But it seems to load configuration from both files, which leads to an error in compiling  
```
info  - Loaded env from C:\Users\addis\SoftwareProjects\gloss-translation\gloss-translation\packages\api\.env.local
info  - Loaded env from C:\Users\addis\SoftwareProjects\gloss-translation\gloss-translation\packages\api\.env
event - compiled client and server successfully in 25.9s (150 modules)
[ ready ] on http://localhost:4300
wait  - compiling /_error (client and server)...
event - compiled client and server successfully in 885 ms (151 modules)
```

Reply:
That should be `DATABASE_URL=postgresql://localhost:5432/db-name`.

You can leave .env alone since .env.local takes precedence. So we can put in .env vars that aren't secret, and .env.local ones that shouldn't be checked in